<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="21" passed="18" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-06-29T15:06:35 IST" name="All Test Suite" finished-at="2022-06-29T15:07:00 IST" duration-ms="25060">
    <groups>
    </groups>
    <test started-at="2022-06-29T15:06:35 IST" name="C:/Users/E-LINE/Desktop/Fiverrr/halsaBabyAPI/src/test/java/userOnboarding" finished-at="2022-06-29T15:07:00 IST" duration-ms="25060">
      <class name="userOnboarding.resendEmailVerification">
        <test-method signature="resendEmailVer()[pri:0, instance:userOnboarding.resendEmailVerification@37911f88]" started-at="2022-06-29T15:06:56 IST" name="resendEmailVer" finished-at="2022-06-29T15:06:57 IST" duration-ms="945" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [400]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [400]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1240)
at org.testng.Assert.assertEquals(Assert.java:1274)
at userOnboarding.resendEmailVerification.resendEmailVer(resendEmailVerification.java:36)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- resendEmailVer -->
        <test-method signature="resendEmailVer_Negative()[pri:1, instance:userOnboarding.resendEmailVerification@37911f88]" started-at="2022-06-29T15:06:57 IST" name="resendEmailVer_Negative" finished-at="2022-06-29T15:06:58 IST" duration-ms="919" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- resendEmailVer_Negative -->
      </class> <!-- userOnboarding.resendEmailVerification -->
      <class name="userOnboarding.updatePhoneNumber">
        <test-method signature="updatePhone()[pri:0, instance:userOnboarding.updatePhoneNumber@23202fce]" started-at="2022-06-29T15:06:52 IST" name="updatePhone" finished-at="2022-06-29T15:06:53 IST" duration-ms="1069" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updatePhone -->
        <test-method signature="updatePhone_Negative()[pri:1, instance:userOnboarding.updatePhoneNumber@23202fce]" started-at="2022-06-29T15:06:53 IST" name="updatePhone_Negative" finished-at="2022-06-29T15:06:54 IST" duration-ms="986" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updatePhone_Negative -->
      </class> <!-- userOnboarding.updatePhoneNumber -->
      <class name="userOnboarding.verifyPhoneNum">
        <test-method signature="verifyPhoneNum()[pri:0, instance:userOnboarding.verifyPhoneNum@31edaa7d]" started-at="2022-06-29T15:06:35 IST" name="verifyPhoneNum" finished-at="2022-06-29T15:06:38 IST" duration-ms="2473" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyPhoneNum -->
        <test-method signature="verifyPhoneNum_Negative()[pri:1, instance:userOnboarding.verifyPhoneNum@31edaa7d]" started-at="2022-06-29T15:06:38 IST" name="verifyPhoneNum_Negative" finished-at="2022-06-29T15:06:39 IST" duration-ms="1550" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyPhoneNum_Negative -->
      </class> <!-- userOnboarding.verifyPhoneNum -->
      <class name="userOnboarding.updateVerify">
        <test-method signature="updateVerify()[pri:0, instance:userOnboarding.updateVerify@7d94beb9]" started-at="2022-06-29T15:06:46 IST" name="updateVerify" finished-at="2022-06-29T15:06:47 IST" duration-ms="1058" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [{
    "timestamp": "2022-06-29T09:36:47.334+0000",
    "status": 400,
    "error": "Bad Request",
    "message": "IsVerified message for this user is already updated",
    "path": "/mobile-backend/api/user/update/is-verified"
}] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [{
    "timestamp": "2022-06-29T09:36:47.334+0000",
    "status": 400,
    "error": "Bad Request",
    "message": "IsVerified message for this user is already updated",
    "path": "/mobile-backend/api/user/update/is-verified"
}] but found [true]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at userOnboarding.updateVerify.updateVerify(updateVerify.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateVerify -->
        <test-method signature="updateVerify_Negative()[pri:1, instance:userOnboarding.updateVerify@7d94beb9]" started-at="2022-06-29T15:06:47 IST" name="updateVerify_Negative" finished-at="2022-06-29T15:06:48 IST" duration-ms="1067" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateVerify_Negative -->
      </class> <!-- userOnboarding.updateVerify -->
      <class name="userOnboarding.verifyEmail">
        <test-method signature="generateToken()[pri:0, instance:userOnboarding.verifyEmail@609bcfb6]" started-at="2022-06-29T15:06:42 IST" name="generateToken" finished-at="2022-06-29T15:06:43 IST" duration-ms="802" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateToken -->
        <test-method signature="verifyEmail()[pri:0, instance:userOnboarding.verifyEmail@609bcfb6]" started-at="2022-06-29T15:06:43 IST" name="verifyEmail" finished-at="2022-06-29T15:06:45 IST" duration-ms="1999" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyEmail -->
        <test-method signature="verifyEmail_Negative()[pri:1, instance:userOnboarding.verifyEmail@609bcfb6]" started-at="2022-06-29T15:06:45 IST" name="verifyEmail_Negative" finished-at="2022-06-29T15:06:46 IST" duration-ms="1004" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyEmail_Negative -->
      </class> <!-- userOnboarding.verifyEmail -->
      <class name="userOnboarding.removePhotoURL">
        <test-method signature="removePhoto()[pri:0, instance:userOnboarding.removePhotoURL@1df8b5b8]" started-at="2022-06-29T15:06:50 IST" name="removePhoto" finished-at="2022-06-29T15:06:51 IST" duration-ms="926" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- removePhoto -->
        <test-method signature="removePhoto_Negative()[pri:1, instance:userOnboarding.removePhotoURL@1df8b5b8]" started-at="2022-06-29T15:06:51 IST" name="removePhoto_Negative" finished-at="2022-06-29T15:06:52 IST" duration-ms="926" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- removePhoto_Negative -->
      </class> <!-- userOnboarding.removePhotoURL -->
      <class name="userOnboarding.createUser">
        <test-method signature="createUser()[pri:0, instance:userOnboarding.createUser@72ade7e3]" started-at="2022-06-29T15:06:39 IST" name="createUser" finished-at="2022-06-29T15:06:41 IST" duration-ms="1672" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [400]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [400]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1240)
at org.testng.Assert.assertEquals(Assert.java:1274)
at userOnboarding.createUser.createUser(createUser.java:45)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createUser -->
        <test-method signature="createUser_Negative()[pri:1, instance:userOnboarding.createUser@72ade7e3]" started-at="2022-06-29T15:06:41 IST" name="createUser_Negative" finished-at="2022-06-29T15:06:42 IST" duration-ms="1022" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createUser_Negative -->
      </class> <!-- userOnboarding.createUser -->
      <class name="userOnboarding.updateEmail">
        <test-method signature="updateEmail()[pri:0, instance:userOnboarding.updateEmail@7b993c65]" started-at="2022-06-29T15:06:54 IST" name="updateEmail" finished-at="2022-06-29T15:06:55 IST" duration-ms="1421" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateEmail -->
        <test-method signature="updateEmail_Negative()[pri:0, instance:userOnboarding.updateEmail@7b993c65]" started-at="2022-06-29T15:06:55 IST" name="updateEmail_Negative" finished-at="2022-06-29T15:06:56 IST" duration-ms="947" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateEmail_Negative -->
      </class> <!-- userOnboarding.updateEmail -->
      <class name="userOnboarding.getCurrentUser">
        <test-method signature="getCurrentUser()[pri:0, instance:userOnboarding.getCurrentUser@6f1c29b7]" started-at="2022-06-29T15:06:58 IST" name="getCurrentUser" finished-at="2022-06-29T15:06:59 IST" duration-ms="908" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getCurrentUser -->
        <test-method signature="getCurrentUser_Negative()[pri:1, instance:userOnboarding.getCurrentUser@6f1c29b7]" started-at="2022-06-29T15:06:59 IST" name="getCurrentUser_Negative" finished-at="2022-06-29T15:07:00 IST" duration-ms="1196" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getCurrentUser_Negative -->
      </class> <!-- userOnboarding.getCurrentUser -->
      <class name="userOnboarding.updateUserProfile">
        <test-method signature="updateUserPro()[pri:0, instance:userOnboarding.updateUserProfile@560348e6]" started-at="2022-06-29T15:06:48 IST" name="updateUserPro" finished-at="2022-06-29T15:06:49 IST" duration-ms="1140" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateUserPro -->
        <test-method signature="updateUserPro_Negative()[pri:1, instance:userOnboarding.updateUserProfile@560348e6]" started-at="2022-06-29T15:06:49 IST" name="updateUserPro_Negative" finished-at="2022-06-29T15:06:50 IST" duration-ms="933" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateUserPro_Negative -->
      </class> <!-- userOnboarding.updateUserProfile -->
    </test> <!-- C:/Users/E-LINE/Desktop/Fiverrr/halsaBabyAPI/src/test/java/userOnboarding -->
  </suite> <!-- All Test Suite -->
</testng-results>
